src/library/dao/playlistdao.cpp

```diff
@@ -1042,3 +1044,24 @@ void PlaylistDAO::sendToAutoDJ(const QList<TrackId>& trackIds, AutoDJSendLoc loc
             break;
     }
 }
+
+
+QList<QPair<int, int> > PlaylistDAO::getPlaylistTracksPositionRowIds(int playlistID) {
```

Comment:
When talking about "track positions" in the PlaylistDAO the prefix "Playlist" is redundant